packages/protocol/contracts/governance/LockedGold.sol

```diff
@@ -721,10 +721,26 @@ contract LockedGold is ILockedGold, ReentrancyGuard, Initializable, UsingRegistr
     list.length = lastIndex;
   }
 
-  function isAccount(address account) internal view returns (bool) {
+  /**
+   * @notice Check if an account already exists.
+   * @param account The address of the account
+   * @return Returns `true` if account exists. Returns `false` otherwise.
+   *         In particular it will return `false` if a delegate with given address exists.
+   */
+  function isAccount(address account) public view returns (bool) {
     return (accounts[account].exists);
   }
 
+  /**
+   * @notice Check if a delegate already exists.
+   * @param account The address of the delegate
+   * @return Returns `true` if delegate exists. Returns `false` otherwise.
+   */
+  function isDelegate(address account) external view returns (bool) {
```

Comment:
Can we get some unit tests for this function?

Reply:
ok